veracruz-server-test/src/main.rs

```diff
@@ -125,16 +120,28 @@ mod tests {
         let rst = NEXT_TICKET.fetch_add(1, Ordering::SeqCst);
 
         SETUP.call_once(|| {
-            info!("SETUP.call_once called");
+            println!("SETUP.call_once called");
             let _main_loop_handle = std::thread::spawn(|| {
                 let mut sys = System::new("Veracruz Proxy Attestation Server");
+                #[cfg(feature = "nitro")]
```

Comment:
Similar comment here as before about maybe using `cfg_if` for these mutually-exclusive feature tests.